apps/meteor/tests/unit/server/startup/initialData.tests.ts

```diff
@@ -0,0 +1,235 @@
+import { expect } from 'chai';
+import { beforeEach, it } from 'mocha';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+
+const getUsersInRole = sinon.stub();
+const checkUsernameAvailability = sinon.stub();
+const validateEmail = sinon.stub();
+const addUserRolesAsync = sinon.stub();
+const models = {
+	Settings: {},
+	Rooms: {},
+	Users: {
+		create: sinon.stub(),
+		findOneByEmailAddress: sinon.stub(),
+	},
+};
+const setPasswordAsync = sinon.stub();
+const settingsGet = sinon.stub();
+
+const { insertAdminUserFromEnv } = proxyquire.noCallThru().load('../../../../server/startup/initialData.js', {
+	'meteor/accounts-base': {
+		Accounts: {
+			setPasswordAsync,
+		},
+	},
+	'meteor/meteor': {
+		Meteor: {
+			startup: sinon.stub(),
+		},
+	},
+	'../../app/authorization/server': {
+		getUsersInRole,
+	},
+	'../../app/file-upload/server': {},
+	'../../app/file/server': {},
+	'../../app/lib/server/functions/addUserToDefaultChannels': {},
+	'../../app/lib/server/functions/checkUsernameAvailability': {
+		checkUsernameAvailability,
+	},
+	'../../app/settings/server': {
+		settings: { get: settingsGet },
+	},
+	'../../lib/emailValidator': {
+		validateEmail,
+	},
+	'../lib/roles/addUserRoles': {
+		addUserRolesAsync,
+	},
+	'@rocket.chat/models': models,
+});
+
+describe('insertAdminUserFromEnv', () => {
+	beforeEach(() => {
+		getUsersInRole.reset();
+		checkUsernameAvailability.reset();
+		validateEmail.reset();
+		addUserRolesAsync.reset();
+		models.Users.create.reset();
+		models.Users.findOneByEmailAddress.reset();
+		setPasswordAsync.reset();
+		settingsGet.reset();
+		process.env.ADMIN_PASS = 'pass';
+	});
+
+	it('should do nothing if process.env.ADMIN_PASS is empty', async () => {
+		process.env.ADMIN_PASS = '';
+		const result = await insertAdminUserFromEnv();
+		expect(getUsersInRole.called).to.be.false;
+		expect(result).to.be.undefined;
```

Comment:
```suggestion
```

I think this is unnecessary since the function doesn't return anything either way.

Reply:
Yup, this is just to test that the function actually ran and returned, as there's only one return.

Kinda a dummy check, but also ensures the function just ran and didn't throw